python/dgl/nn/pytorch/conv/appnpconv.py

```diff
@@ -45,17 +45,17 @@ class APPNPConv(nn.Module):
     >>> feat = th.ones(6, 10)
     >>> conv = APPNPConv(k=3, alpha=0.5)
     >>> res = conv(g, feat)
-    >>> res
-    tensor([[1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000,
-            1.0000],
-            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000,
-            1.0000],
-            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000,
-            1.0000],
-            [1.0303, 1.0303, 1.0303, 1.0303, 1.0303, 1.0303, 1.0303, 1.0303, 1.0303,
-            1.0303],
-            [0.8643, 0.8643, 0.8643, 0.8643, 0.8643, 0.8643, 0.8643, 0.8643, 0.8643,
-            0.8643],
+    >>> print(res)
```

Comment:
@Rhett-Ying By the way, dropout is non-deterministic and it's natural to get different results from multiple calls.

Reply:
oh, I see. thanks...